pipelines/upgrade_pipeline.yml

```diff
@@ -0,0 +1,65 @@
+- hosts: localhost
+  vars_files:
+    - vars/{{ pipeline_type }}_base.yml
+    - vars/{{ pipeline_type }}_upgrade_base.yml
+  roles:
+    - forklift
+
+- hosts: pipeline-upgrade-{{ pipeline_type }}-{{ pipeline_version}}-{{ pipeline_os }}
+  become: yes
+  vars:
+  vars_files:
+    - vars/{{ pipeline_type }}_base.yml
+    - vars/{{ pipeline_type }}_base_release.yml
+    - vars/{{ pipeline_type }}_upgrade_{{ pipeline_version }}.yml
+    - vars/{{ pipeline_type }}_{{ pipeline_version_start }}.yml
+  roles:
+    - umask
+    - selinux
+    - etc_hosts
+    - foreman_server_repositories
+    - role: foreman_client_repositories
+      when: pipeline_version_start == "nightly" or pipeline_version_start is version('3.9', '>=')
```

Comment:
3.9 sounds like a Katello version. I think we started with 1.20. We should always have the foreman version at this point so let's normalize on that. This is repeated a few times so I won't duplicate the comment :)

Reply:
that's fair, yeah.

Reply:
I'll fix that as soon as https://github.com/theforeman/forklift/pull/929 is in.

Reply:
done via #929